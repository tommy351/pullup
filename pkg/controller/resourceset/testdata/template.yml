---
apiVersion: pullup.dev/v1alpha1
kind: ResourceSet
metadata:
  name: test-46
  namespace: test
  uid: c37ee64f-5c69-4c53-8585-adc137d0acca
spec:
  base:
    ref: master
    sha: efef1cefcd199ce6e0f1f0b9f8acac420563044b
  head:
    ref: fix-test
    sha: aad2b388e813f3a3c90672332bdd2f9c5bac568c
  number: 46
  resources:
    - apiVersion: v1
      kind: Pod
      metadata:
        name: test
      spec:
        containers:
          - name: '{{ .Name }}'
            image: 'gcr.io/test/foo:{{ .Spec.Head.SHA }}'
            env:
              - name: PR_NUMBER
                value: '{{ .Spec.Number }}'
