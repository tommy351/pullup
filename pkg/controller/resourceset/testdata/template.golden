# This is a golden file. DO NOT EDIT.
/v1, Kind=Pod:
  test/test-46:
    apiVersion: v1
    kind: Pod
    metadata:
      name: test-46
      namespace: test
      ownerReferences:
      - apiVersion: pullup.dev/v1alpha1
        blockOwnerDeletion: true
        controller: true
        kind: ResourceSet
        name: test-46
        uid: c37ee64f-5c69-4c53-8585-adc137d0acca
    spec:
      containers:
      - env:
        - name: PR_NUMBER
          value: "46"
        image: gcr.io/test/foo:aad2b388e813f3a3c90672332bdd2f9c5bac568c
        name: test-46
